packages/builder/src/pages/builder/app/[application]/design/[screenId]/screens/_components/NewScreenModal.svelte

```diff
@@ -23,71 +25,84 @@
     onConfirm={confirmScreenSelection}
     {onCancel}
     disabled={!selectedScreenMode}
-    size="L"
+    size="M"
   >
     <Layout noPadding gap="S">
       <div
-        class="screen-type item"
+        class="screen-type item blankView"
         class:selected={selectedScreenMode == blankScreenModeKey}
         on:click={() => {
           selectedScreenMode = blankScreenModeKey
         }}
       >
         <div class="content screen-type-wrap">
-          <Icon name="WebPage" />
+          <img
+            alt="blank screen preview"
+            class="preview"
+            src={blankScreenPreview}
+          />
           <div class="screen-type-text">
             <Heading size="XS">Blank screen</Heading>
-            <Body size="S">Add a blank screen</Body>
+            <Body size="S">Add an empty blank screen</Body>
           </div>
         </div>
         <div
           style="color: var(--spectrum-global-color-green-600); float: right"
         >
-          {#if selectedScreenMode == blankScreenModeKey}
-            <div class="checkmark-spacing">
-              <Icon size="S" name="CheckmarkCircle" />
-            </div>
-          {/if}
+          <div
+            class={`checkmark-spacing ${
+              selectedScreenMode == blankScreenModeKey ? "visible" : ""
```

Comment:
You don't need to change this, but just a FYI that the usual (and quite nice) way to conditionally apply classes like this in svelte is:
```html
<div
  class="checkmark-spacing"
  class:visible={selectedScreenMode == blankScreenModeKey}
  ....others
/>
```
Keeps it readable and works perfectly for multiple classes.
NAB but just letting you know for the future!

Reply:
Ah, wasn't aware you could combine the two, thanks for the heads up.